community-operators/sysdig/sysdig-operator.v1.4.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: sysdig-operator.v1.4.0
+  namespace: placeholder
+  annotations:
+    capabilities: "Basic Install"
+    categories: "Security, Monitoring"
+    certified: "false"
+    description: |-
+      [Sysdig](https://www.sysdig.com/) is a unified platform for container and microservices monitoring, troubleshooting, security and forensics. Sysdig platform has been built on top of [Sysdig tool](https://sysdig.com/opensource/sysdig/) and [Sysdig Inspect](https://sysdig.com/blog/sysdig-inspect/) open-source technologies.
```

Comment:
This description shouldn't use markdown since it won't be rendered in the UI. `spec.description` can use markdown.